features/step_definitions/promotional_features_steps.rb

```diff
@@ -158,10 +169,19 @@
 end
 
 Then(/^the promotional features should be in the following order:$/) do |promotional_feature_list|
-  promotion_feature_ids = all(".promotional_feature").map { |element| element[:id] }
+  if using_design_system?
```

Comment:
This page only ever uses the [GOV.UK Design System](https://github.com/alphagov/whitehall/pull/7644/files#diff-1de1f6ca4df7f1675999edf0f4c9af90bbcfcd1c001d84a5d134c68859ff3a20L67) so i think we just need one bit of logic that works here.

Reply:
If i put logic for preview design system then its failing in default mode